### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewProblems_ScreenReader: NVDA does not read the selected text when we switch to Editor.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52007]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10
MAC Version: HighSierra10.13.5 

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Problems.
3. Tab till the error generated and click on it .

Actual:
When we switch to Editor to resolve the error, NVDA does not read the texts which are selected in editor.

Expected:
NVDA should read the selected text.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[[A11y VSCode Problems NVDA not reading selection.pptx](https://github.com/Microsoft/vscode/files/2105577/A11y.VSCode.Problems.NVDA.not.reading.selection.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This works perfectly fine for me in the latest nvda and vscode insiders.
![screen shot 2018-07-04 at 11 17 38](https://user-images.githubusercontent.com/1926584/42268458-dfb9fa9a-7f7b-11e8-8ae5-316d435197ea.png)
